applications/integrationtests/geometryConcrete/GmshWriter.cpp

```diff
@@ -0,0 +1,83 @@
+#include "BoostUnitTest.h"
+
+#include <boost/filesystem.hpp>
+#include <cstdlib> // std::system
+#include "geometryConcrete/GmshWriter.h"
+#include "mechanics/mesh/MeshGmsh.h"
+
+using namespace NuTo;
+
+void CheckMesh(std::string meshFile, GmshWriter::Options opts)
+{
+    MeshGmsh meshGmsh(meshFile);
+    BOOST_CHECK(not meshGmsh.GetPhysicalGroup(opts.physicalGroupMatrix).Empty());
+    BOOST_CHECK(not meshGmsh.GetPhysicalGroup(opts.physicalGroupAggregates).Empty());
+
+    if (opts.interfaceThickness != 0)
+        BOOST_CHECK(not meshGmsh.GetPhysicalGroup(opts.physicalGroupInterfaces).Empty());
+}
+
+std::string GmshPath()
+{
+    return boost::unit_test::framework::master_test_suite().argv[1];
```

Comment:
Why are you passing this gmsh path around? Can we not assume that it lives in `$PATH`?

Reply:
Not sure. I use this cmake [`find_proram`](https://cmake.org/cmake/help/v3.0/command/find_program.html) to find the executable. By setting `CMAKE_FIND_ROOT_PATH`, you can specify additional search paths. This seemed more like the "cmake" way.